.travis.yml

```diff
@@ -1,15 +1,15 @@
 language: nix
 
-env:
-  NIX_PATH=nixpkgs=https://github.com/NixOS/nixpkgs/archive/c0b1e8a5fb174cd405dcca9f7fec275714ad9f4b.tar.gz
-
 install:
+  - source scripts/set_nixpath.sh
   - nix-env -iA nixopsUnstable -f '<nixpkgs>'
```

Comment:
@cleverca22, the way I've done this was:

- https://github.com/input-output-hk/iohk-nixops/blob/devops-123-refactor/shell.nix#L5, and
- https://github.com/input-output-hk/iohk-nixops/blob/devops-123-refactor/shell.nix#L45